cmake/projects/cryptopp/hunter.cmake

```diff
@@ -0,0 +1,32 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+# Load used modules
+include(hunter_add_version)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+# List of versions here...
+hunter_add_version(
+        PACKAGE_NAME
+        cryptopp
+        VERSION
+        "5.6.5"
+        URL
+        "https://github.com/weidai11/cryptopp/archive/CRYPTOPP_5_6_5.tar.gz"
+        SHA1
+        3aa4f2fcfee05ed304c6614325c90eba0dc75f16
+)
+
+hunter_add_version(
+        PACKAGE_NAME
+        cryptopp
+        VERSION
+        "5.6.4"
+        URL
+        "https://github.com/weidai11/cryptopp/archive/CRYPTOPP_5_6_4.tar.gz"
+        SHA1
+        066e01e704bbdc0f4f785f344c05c8259078a020
+)
+
```

Comment:
Is it [cacheable](https://docs.hunter.sh/en/latest/reference/user-modules/hunter_cacheable.html)?